plugins/signals-backend/src/service/standaloneServer.ts

```diff
@@ -62,6 +64,14 @@ export async function startStandaloneServer(
       mockSubscribers.push(subscription);
     },
   };
+  const tokenManager = ServerTokenManager.fromConfig(config, {
+    logger,
+  });
+
+  const permissions = ServerPermissionClient.fromConfig(config, {
+    discovery,
+    tokenManager,
+  });
```

Comment:
i guess these get replaced when adding a `dev/index.ts`

Reply:
Yeah, another PR for that